packages/graphql/src/translate/create-update-and-params.ts

```diff
@@ -51,25 +51,35 @@ function createUpdateAndParams({
     parentVar,
     chainStr,
     insideDoWhen,
-    withVars,
+    withProjector,
     context,
     parameterPrefix,
-    fromTopLevel,
 }: {
     parentVar: string;
     updateInput: any;
     varName: string;
     chainStr?: string;
     node: Node;
-    withVars: string[];
+    withProjector: WithProjector,
     insideDoWhen?: boolean;
     context: Context;
     parameterPrefix: string;
-    fromTopLevel?: boolean;
 }): [string, any] {
     let hasAppliedTimeStamps = false;
 
+    /**
+     * **NOTE**
+     * Update input is now being sent to neo4j twice. Instead of flattening each input parameter,
+     * perhaps we can utilize the object being sent to neo4j?
+     * `objUpdateParams` is necessary for subscriptions to sent an object containing
+     * the updated properties.
+     */
+    const objUpdateParams = {}; 
+    const objUpdateParamsKey = chainStr || `${parentVar}_update`;
```

Comment:
please advise:
```
     * Update input is now being sent to neo4j twice. Instead of flattening each input parameter,
     * perhaps we can utilize the object being sent to neo4j?
     * `objUpdateParams` is necessary for subscriptions to sent an object containing
     * the updated properties.
```

Reply:
Sorry @KleptoKat, missed this one. We can definitely pass to Neo4j as an object - feel free to clutter the Cypher parameter somewhat because this is an area we need to clean anyway.